services/frontend-service/pkg/config/config.go

```diff
@@ -31,7 +31,7 @@ type ServerConfig struct {
 	AzureRedirectUrl    string `default:"" split_words:"true"`
 	Version             string `default:""`
 	SourceRepoUrl       string `default:"" split_words:"true"`
-	AllowedOrigins      string `default:"" split_words:"true"`
+	AllowedOrigins      string `default:"localhost:*" split_words:"true"`
```

Comment:
Is that acutally the right place for this default? We only need it on localhost, so maybe rather put it in the makefile?

Reply:
yes, you are right. I am moving it to the makefile